src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -392,6 +371,19 @@
       <Link>Common\Unix\System.Native\Interop.GetTimestamp.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true' AND '$(IsPartialFacadeAssembly)'!='true' AND '$(TargetGroup)' != 'uap'">
```

Comment:
This condition can be just `'$(TargetGroup)' == 'netcoreapp' AND '$(TargetsWindows)' == 'true'`